c_api/build.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 Adobe. All rights reserved.
```

Comment:
Is the copyright year off by a few years? File seems new?

Reply:
Yes, it is, but clippy has decided that every file must have the same copyright date or it will complain bitterly and break our builds. I think that's very wrong policy, but I haven't been fighting it for now.